format/index.md

```diff
@@ -753,25 +758,21 @@ GET /articles/1?include=author,comments,comments.author
 
 ### Sparse Fieldsets <a href="#fetching-sparse-fieldsets" id="fetching-sparse-fieldsets" class="headerlink"></a>
 
-A client **MAY** request that an endpoint return only specific
-[fields](#document-structure-resource-fields) in the response on a per-type basis
-by including a `fields[TYPE]` parameter. The value of the parameter is a
-comma-separated (U+002C COMMA, ",") list that refers to the names(s) of the
-fields to be returned.
+A client **MAY** request that an endpoint return only specific [fields] in the
+response on a per-type basis by including a `fields[TYPE]` parameter.
+
+> Note: Only [fields] are affected; `type`, `id`, and `self` are included as normal.
```

Comment:
since `self` links aren't required, then perhaps we should say "optionally `self` links" instead of just "`self`"?